examples/00_tutorials/plot_single_subject_single_run.py

```diff
@@ -147,86 +136,103 @@
 # The first column contains the expected response profile of regions which are
 # sensitive to the auditory stimulation.
 # Let's plot this first column
+import matplotlib.pyplot as plt
 
-plt.plot(design_matrix["active"])
+plt.plot(design_matrix["listening"])
 plt.xlabel("scan")
 plt.title("Expected Auditory Response")
-plt.show()
+
+show()
+
 
 # %%
 # Detecting voxels with significant effects
 # -----------------------------------------
 #
-# To access the estimated coefficients (Betas of the :term:`GLM` model), we
-# created :term:`contrast` with a single '1' in each of the columns: The role
-# of the :term:`contrast` is to select some columns of the model --and
-# potentially weight them-- to study the associated statistics. So in
-# a nutshell, a :term:`contrast` is a weighted combination of the estimated
-# effects.  Here we can define canonical contrasts that just consider
-# the two effects in isolation ---let's call them "conditions"---
-# then a :term:`contrast` that makes the difference between these conditions.
+# To access the estimated coefficients (Betas of the :term:`GLM` model),
+# we created :term:`contrast` with a single '1' in each of the columns:
+# The role of the :term:`contrast` is to select some columns of the model
+# --and potentially weight them-- to study the associated statistics.
+# So in a nutshell, a :term:`contrast`
+# is a weighted combination of the estimated effects.
+# Here we can define canonical contrasts that just consider
+# the effect of the stimulation in isolation.
+#
+# .. note::
+#
+#       Here the baseline is implicit, so passing a value of 1
+#       for the first column will give contrast for: ``listening > rest``
+#
 
 import numpy as np
 
-conditions = {"active": np.zeros(16), "rest": np.zeros(16)}
-conditions["active"][0] = 1
-conditions["rest"][1] = 1
+nb_regressors = design_matrix.shape[1]
+activation = np.zeros(nb_regressors)
```

Comment:
I'm not familiar the with `nb_` convention. Is that `number`? I typically use `n_`.

Reply:
yeah but you are right, `n_` is more frequent (at least in the nilearn codebase)

```bash
$ grep -nr " nb_" nilearn/**/*.py | wc -l
45

$ grep -nr " n_" nilearn/**/*.py | wc -l 
2044
```

probably worth cleaning the other instances to be more internally consistent